PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+High level description including:
+ - What this change is enabling / fixing.
+ - High level description of how things changed
+
+# Issues
+
+[DCOS-<number>](https://mesosphere.atlassian.net/browse/DCOS-<number>)
+...
+
+# Checklist
+
+ - [ ] Included a test which will fail if code is reverted but test is not
```

Comment:
The revert part is critical to what's being asked for. People will always say they "tested it in their component", the goal is to get across we want to test the _integration_ which is distinct from testing that the feature works at all (which unit and integration tests inside their component cover)